modules/tinymce/src/core/main/ts/content/ContentTypes.ts

```diff
@@ -22,6 +22,7 @@ export interface SetContentArgs {
   paste?: boolean;
   load?: boolean;
   initial?: boolean;
+  [key: string]: any;
```

Comment:
Does this end up as a public api change. Then maybe that needs to be logged since that does affect what types of events you can pass into the content apis. I wonder if we even want to be relaxed about this. The code allows anything but that is maybe more of a side effect of how this is implemented.

Reply:
What are the implications of being strict here? I do not think that passing custom keys can break something, but it can satisfy integrators' and plugin-developers' needs. Maybe it makes sense to have similar behavior in both `GetContentArgs` and `SetContentArgs`?

Reply:
The problem could be that we then invite people to add custom things here that might clash with things we want to add our self later. But since we have no runtime validation of this and it can be used in JS projects I guess it's already very relaxed here. But if this is a public api change we need a changelog of this and not the fact that we added tests. Something that describes that the types now accept custom properties.